packages/react-jss/src/createHoc.js

```diff
@@ -3,9 +3,10 @@
 import React, {Component, type ComponentType} from 'react'
 import PropTypes from 'prop-types'
 import {ThemeContext} from 'theming'
-import {getDynamicStyles, SheetsManager, type StyleSheet, type Classes} from 'jss'
+import {getDynamicStyles, SheetsManager, type StyleSheet} from 'jss'
+import memoize from 'memoize-one'
```

Comment:
do we need a lib for this? I have a feeling that function is 2-3LOC and easier to inline than to have a dependency